### PR TITLE
Fix the path to the lock files in push.py

### PR DESCRIPTION
**Pull request by puiterwijk**
Signed-off-by: Patrick Uiterwijk <patrick@puiterwijk.org>

### CODE REVIEW

**Comment by puiterwijk** [comment]
I considered keeping the `--staging` flag as a no-op for backwards compatibility, but given that we're making this a Bodhi 3.0 anyway, I figured I'd just do away with it.

File: bodhi/server/push.py [hunk 1]
```diff
@@ -42,21 +42,15 @@
...
-    staging = kwargs.pop('staging')
     resume = kwargs.pop('resume')
 
     lockfiles = defaultdict(list)
     locked_updates = []
-    if staging:
-        locks = '/var/cache/bodhi/mashing/MASHING-*'
-    else:
-        locks = '/mnt/koji/mash/updates/MASHING-*'
+    locks = '%s/MASHING-*' % config.get('mash_dir')
```
Comment (bowlofeggs) [inline bodhi/server/push.py:53]: Yeah good call on killing ```--staging``` - since ```mash_dir``` is part of staging's config, it's not needed anyway.

**Review by bowlofeggs — APPROVED** [review]

File: bodhi/server/push.py [hunk 2]
```diff
@@ -42,21 +42,15 @@
               help='Push updates with a specific request (default: testing,stable)')
 @click.option('--resume', help='Resume one or more previously failed pushes',
               is_flag=True, default=False)
-@click.option('--staging', help='Use the staging bodhi instance',
-              is_flag=True, default=False)
```
Comment (bowlofeggs) [inline bodhi/server/push.py:46]: Can we add a note to ```docs/release_notes.rst``` about this?

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

**Review by bowlofeggs — APPROVED** [review]

**Comment by puiterwijk** [comment]
Release notes changes look good. :+1:

**Comment by bowlofeggs** [comment]
jenkies test

**Comment by bowlofeggs** [comment]
In the interest of time, @puiterwijk and I have agreed that running the ```devel/run_tests.sh``` script locally would be an acceptable subtitute for waiting on CI to pass this build. The tests pass, and there is minimal risk that Jenkies would disgree so I am going to merge this.

**Comment by bowlofeggs** [comment]
This was cherry picked to the 3.0 branch as c95ba78580c3b70224146c4752c387319d9e6eab.

**Pull request merged**